src/components/Home.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Home() {
+  return (
+    <>
+      <main>
+        <div className="header">
+          <h1>Math Magicians</h1>
+          <nav>
+            <Link to="/about">Quote</Link>
+            <Link to="/">Home</Link>
+            <Link to="/Calculator">Calculator</Link>
+          </nav>
+        </div>
```

Comment:
- [ ] [Optional] [Highly recommended] Please wrap your header inside a `<header>` tag instead of `<div>` tag for semantic reasons.
- [ ] [Optional] [Highly recommended] Instead of repeating your header in every component make a `Header` component and add it in `App.js` just above your `routes` directly to keep your code DRY.

Reply:
I already implemented your great suggestions. Much appreciated @Omar-Muhamad